app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -74,14 +77,14 @@ case class TestAgent(override val userId: String,
   override val affinityGroup = "Agent"
 }
 
-case class TestIndividualCreatedResponse(userId: String, password: String, saUtr: Option[SaUtr], nino: Option[Nino], mtdItId: Option[MtdItId])
+case class TestIndividualCreatedResponse(userId: String, password: String, individualDetails: IndividualDetails, saUtr: Option[SaUtr], nino: Option[Nino], mtdItId: Option[MtdItId])
```

Comment:
It looks like `TestIndividualResponse` is not used at all. There is a case class and a companion object with a `TestIndividualResponse.from(TestIndividual)` function and that is not used either. I'll remove it completely to avoid confusion.